cypress/integration/basic_tests.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+
+
+
+describe('Basic Login and Signup', function() {
+  beforeEach(function() {
+    // reset the database prior to every test
+    cy.resetDatabase()
+
+    // Create default admin user
```

Comment:
Perhaps basic functionality tests should be done as a non-admin? One of the ways we might plausibly mess up and not notice, is by messing up the permissions so that something that non-admins do starts requiring admin

Reply:
Yep, agree. We probably actually want to create two users, one admin one non-admin and use the non-admin for most checks.